### PR TITLE
Portage/Ebuild.hs: move S= to right after MY_P

### PR DESCRIPTION
**Pull request by ezzieyguywuf**
I was unable to actually test this, because I'm not sure how to trigger this
section of the code generator.

It compiles, though, and the test suite passes.

### CODE REVIEW

File: Portage/EBuild.hs [hunk 1]
```diff
@@ -125,6 +125,9 @@ showEBuild now ebuild =
      Nothing -> id
      Just pn -> ss "MY_PN=". quote pn. nl.
                 ss "MY_P=". quote "${MY_PN}-${PV}". nl. nl).
+  (case my_pn ebuild of
```
Comment (trofi) [inline Portage/EBuild.hs:128]: I think you can move `ss "S=". quote ("${WORKDIR}/${MY_P}").` right under case one line above.

Comment (ezzieyguywuf) [reply Portage/EBuild.hs:128]: Yes you're right! I will fix this

Comment (ezzieyguywuf) [reply Portage/EBuild.hs]: Ok this is done now.

**Comment by trofi** [comment]
Pulled as is. Thank you!

**Pull request merged**